pages/agent/v3/upgrading.md.erb

```diff
@@ -79,17 +79,17 @@ steps:
 
 Previously we didn't support variable interpolation like `${MY_VARIABLE_NAME}` or `$MY_VARIABLE_NAME`, so if you have any of these in your previous pipeline's and expect the literal text to render, you will need to escape the dollar signs like `$$MY_VARIABLE_NAME`.
 
-### Default flags for `git clean` include `-x`
+### Checkout clean no longer ignores files in `.gitignore`
 
 Older agents didn't remove files from your working directory that were ignored by git. The new default values for git clean are `-fxdq`. If you've previously overridden your `git-clean-flags` in your config, it might be a good chance to comment them out and use the standard behaviour.
 
-## Upgrading a 2.0 agent
+## Upgrading from a 2.0 agent
 
-To upgrade you simply install the a new 3.0 agent using the [standard installation methods](/docs/agent/v2/installation). You can use agent tags to create a new queue for 3.0 builds so that you can test it in parallel to your existing agents.
+To upgrade you simply install the a new 3.0 agent using the [standard installation methods](installation). You can use agent tags to create a new queue for 3.0 builds so that you can test it in parallel to your existing agents.
 
 To make installation easy we’ve created packages for each of the major operating systems.
 
-## Upgrading a 3.0 beta agent
+## Upgrading from beta of a 3.0 agent
```

Comment:
Tiny suggestion, feel free to ignore - this might flow better as 'Upgrading from a beta to the stable 3.0 agent'.

Reply:
Yup, much better.

Reply:
Hmmm. I made a change, but not quite the one you suggested. What we are describing here is specifically upgrading from a 3.0-beta to a 3.0-stable. I reckon what you suggested sounds more like any beta to a 3.0 stable. I dunno actually, I guess the same instructions would apply either way.

Reply:
The change sounds good!